test_vm/vagrant_config_defaults.rb

```diff
@@ -2,6 +2,6 @@
   {nfs: true, nfs_version: 4, mount_options: ["noatime", "nodiratime"]}
 
 # Default options
-CONFIG['memory'] = "1100"
+CONFIG['memory'] = "2048"
```

Comment:
If Oracle+DB2+Informix are started altogether even 2GB won't be enough. My considerations in terms of keeping the default small were that it seemed less risky to run out of memory inside the VM than in the host OS.